src/Plugin/Field/FieldFormatter/KaizenFormatter.php

```diff
@@ -69,23 +69,26 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
     $elements = [];
-    $attributes =  new Attribute($this->pluginDefinition['variables']['attributes'] ? $this->pluginDefinition['variables']['attributes'] : []);
+    $attributes = new Attribute($this->pluginDefinition['variables']['attributes'] ? $this->pluginDefinition['variables']['attributes'] : []);
     $attributes->addClass($this->getSetting('modifiers'));
     $attributes->addClass($this->getSetting('extra_classes'));
+    list(, $formatter_id) = explode(":", $this->getPluginId());
```

Comment:
instead of `explode()` please use `getBaseId()` or `getDerivativeId()` (I bet later is what you need)

because implementation should not care about derivative separator, see `\Drupal\Component\Plugin\PluginBase::getDerivativeId`